assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -2095,15 +2095,15 @@ protected String build_student_view_grade_context(VelocityPortlet portlet, Conte
 			}
 			context.put("submission", submission);
 			
-                        if (assignment.isGroup()) {
-                            String _grade_override= submission.getGradeForUser(UserDirectoryService.getCurrentUser().getId());
-                            if (_grade_override != null) {
-                                if (assignment.getContext() != null && assignment.getContent().getTypeOfGrade() == 3)
-                                    context.put("override", displayGrade(state, _grade_override));
-                                else
-                                    context.put("override", _grade_override);
-                            }
-                        }
+            if (assignment.isGroup()) {
+                String _grade_override= submission.getGradeForUser(UserDirectoryService.getCurrentUser().getId());
+                if (_grade_override != null) {
+                    if (assignment.getContext() != null && assignment.getContent().getTypeOfGrade() == 3)
```

Comment:
Since you're refactoring this block, it would be nice to add braces to the if/else here